manifests/storage-modules.yaml

```diff
@@ -51,3 +51,25 @@ parameters:
     value: 30
   - name: removal-policy
     value: DESTROY
+---
+name: opensearch
+path: git::https://github.com/awslabs/idf-modules.git//modules/storage/opensearch/
```

Comment:
We should lock this version to a release of IDF:
```
git::https://github.com/awslabs/idf-modules.git//modules/storage/opensearch?ref=release/1.7.0&depth=1
```

Reply:
updated the manifest